src/uthash.h

```diff
@@ -1098,6 +1191,9 @@ typedef struct UT_hash_handle {
    void *key;                        /* ptr to enclosing struct's key  */
    unsigned keylen;                  /* enclosing struct's key len     */
    unsigned hashv;                   /* result of hash-fcn(key)        */
+#if HASH_NOMEM_OK == 1
+   int mem_failed;
+#endif
```

Comment:
Is it OK if we simply state that if `elem.hh.tbl` is 0 after an add operation, there was a memory failure? That sounds consistent to me.